docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,206 @@
+.. _external_storage:
+
+External Storage in Renku Sessions
+==================================
+
+Renku supports accessing data from external storage services like
+S3, Azure Blob, WebDAV, etc. in interactive sessions. Access to external 
+storage can be configured via project settings. 
+
+In the sections below we describe how to add and manage storage configuration to your projects
+and highlight some current limitations:
+
+.. contents::
+  :local: 
+
+.. note::
+
+  Note that this feature is optional; it is enabled at
+  `renkulab.io <http://renkulab.io>`_  but other Renku deployments might not
+  support it. Contact your administrator for more details.
+
+.. warning::
+
+  We support both Read-only and Read/Write access modes.
+  Please note that explicitly marking an attached storage as Read-only is
+  **strongly** encouraged every time you don't have write permissions. This
+  prevents unexpected error messages when trying to access the storage.
+
+Adding external storage to a project
+------------------------------------
+
+Currently, we support configuring external storage access per project.
+Under the project's  ``Settings`` tab, you can find the ``Cloud Storage``
+section. From there, you can click on the ``Add Cloud Storage`` button and follow the
+instructions to go through the guided procedure.
+
+.. image:: ../../_static/images/cloud_storage_1.png
+  :align: center
+  :alt: Cloud Storage section in the project settings
+
+Advanced Mode
+~~~~~~~~~~~~~
+
+If you are familiar with `Rclone <https://rclone.org/>`_, you can switch to
+the "Advanced mode" and copy/paste from any Rclone configuration file.
+
+Mind that we require some additional information that cannot be inferred from
+the configuration file, such as the storage name, the access mode, and the mount
+point for the sessions. 
+
+Step-by-step mode
+~~~~~~~~~~~~~~~~~
+
+This is the default mode. It will guide you through the process of adding an
+external storage to use in your sessions.
+
+Depending on the storage service, the list of options might be long, especially
+when clicking the "Show full list" switch. Most of the time, the basic options
+are sufficient. For more information, please refer to either
+the `Rclone documentation <https://rclone.org/docs/>`_ or the documentation from
+the storage provider you are using.
+
+Specific storage providers
+--------------------------
+
+The backend supports a variety of storage providers. It is based on `Rclone <https://rclone.org/>`_ 
+and we therefore support `most` of the storage services that Rclone supports, except for 
+anything requiring the OAuth flow. The experience is optimized for the most commonly used services, such
```

Comment:
**praise:** 
```suggestion
anything requiring the OAuth flow or virtual rclone providers. The experience is optimized for the most commonly used services, such
```
we don't support e.g. https://rclone.org/local/ or https://rclone.org/memory/, the full list is here: https://github.com/SwissDataScienceCenter/renku-data-services/blob/main/components/renku_data_services/storage/rclone.py#L20-L31